css/style.css

```diff
@@ -1,79 +1,242 @@
-/*
-Project Name: Blank Template
-Client: Your Client
-Author: Your Name
-Developer @GA in NYC
-*/
-
-
-/******************************************
-/* SETUP
-/*******************************************/
-
-/* Box Model Hack */
-* {
-     -moz-box-sizing: border-box; /* Firexfox */
-     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
-     box-sizing: border-box; /* IE */
+  /*
+  Project Name: Blank Template
+  Client: Your Client
+  Author: Your Name
+  Developer @GA in NYC
+  */
+
+  /*Command plus forward slash will comment the line*/
+
+  /******************************************
+  /* SETUP
+  /*******************************************/
+
+  /* Box Model Hack */
+  * {
+       -moz-box-sizing: border-box; /* Firexfox */
+       -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+       box-sizing: border-box; /* IE */
+  }
+
+  /* Clear fix hack */
+  .clearfix:after {
+       content: ".";
+       display: block;
+       clear: both;
+       visibility: hidden;
+       line-height: 0;
+       height: 0;
+  }
+
+  .clear {
+    clear: both;
+  }
+
+  .alignright {
+    float: right;
+    padding: 0 0 10px 10px; /* note the padding around a right floated image */
+  }
+
+  .alignleft {
+    float: left;
+    padding: 0 10px 10px 0; /* note the padding around a left floated image */
+  }
+
+  /******************************************
+  /* BASE STYLES
+  /*******************************************/
+
+  body {
+       color: #000;
+       font-size: 24px;
+       font-weight: 600;
+       line-height: 1.4;
+       font-family: 'Open Sans', sans-serif;
+       background-color: #f0efef;
+       color: #033048;
+       border:solid #033048 1px;
+       padding:5px;
+       margin: auto;
+       width: 960px;
```

Comment:
If you really don't like the page going full width, i instead recommend setting a `max-width` to something like `1024px` or some number that looks good to you. In which case, keep the `margin: auto;`. Your call.